prow/config/plugins.yaml

```diff
@@ -1110,47 +1040,6 @@ external_plugins:
       events:
         - pull_request
         - issue_comment
-    - name: ti-community-autoresponder
-      events:
-        - issue_comment
-        - pull_request_review_comment
-        - pull_request_review
-  pingcap/tidb-binlog:
-    - name: ti-community-lgtm
-      events:
-        - pull_request_review
-        - pull_request
-    - name: ti-community-merge
-      events:
-        - issue_comment
-        - pull_request_review_comment
-        - pull_request
-    - name: needs-rebase
-      events:
-        - pull_request
-        - issue_comment
-    - name: ti-community-tars
-      events:
-        - issue_comment
-        - push
-    - name: ti-community-contribution
-      events:
-        - pull_request
-    - name: ti-community-label
-      events:
-        - issue_comment
-    - name: ti-community-label-blocker
-      events:
-        - pull_request
-    - name: ti-community-cherrypicker
-      events:
-        - pull_request
-        - issue_comment
```

Comment:
The revert operation contains parts that should not be deleted.

Reply:
Thanks for your review! Yes I think it is a mistake when resolve conflict. Updated.